python/selfie-lib/selfie_lib/CommentTracker.py

```diff
@@ -39,8 +67,10 @@ def pathsWithOnce(self) -> Iterable[TypedPath]:
                 if comment == WritableComment.ONCE
             ]
 
-    def hasWritableComment(self, call: CallStack, layout: SnapshotFileLayout) -> bool:
-        path = layout.sourcePathForCall(call)
+    def hasWritableComment(self, call, layout):
+        from selfie_lib.WriteTracker import CallStack
+
+        path = layout.sourcePathForCall(call.location)
```

Comment:
I don't understand this change - in the before we had static types, now they're gone and we have a mid-file import.